packages/keys/assets/ed25519/private_key.pem

```diff
@@ -0,0 +1,3 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
These are test assets, right? I'm wondering if we could put them in some separate place. I would expect that folder `<sub-project>/assets` contains files that are needed for the actual compilation and runtime of the project.
This can be done in a later PR though.